PublicAPI/Microsoft.ApplicationInsights.dll/net45/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,59 @@
+Microsoft.ApplicationInsights.Channel.SamplingScoreGenerator
```

Comment:
Please hide all methods into extension method like this:
https://github.com/microsoft/ApplicationInsights-dotnet/blob/39a5ef23d834777eefdd72149de705a016eb06b0/src/Microsoft.ApplicationInsights/Extensibility/Implementation/TelemetryContextExtensions.cs

Most of them are not designed for end user to use

Reply:
Hid Telemetry Configuration methods, not sure on SamplingScoreGenerator as it's in Channel namespace and has two static methods.